cpp/src/copying/copying_ops.cu

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2018, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "cudf.h"
+
+
+/**
+ * @brief Operations for copying from one column to another
+ * @file copying_ops.cu
+ */
+
+/** 
+ * @brief Scatters the values of a source column into a destination column.
```

Comment:
Do you think we should maybe not duplicate the whole doc comment in the header and source? It's two things to maintain instead of one.